Workbooks/Azure Active Directory/Sign-ins Overview/SigninsOverview.workbook

```diff
@@ -0,0 +1,284 @@
+                  "createdTime": "2018-11-13T19:33:10.164Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 43200000,
+                  "createdTime": "2018-11-13T19:33:10.164Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 86400000,
+                  "createdTime": "2018-11-13T19:33:10.165Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 172800000,
+                  "createdTime": "2018-11-13T19:33:10.166Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 259200000,
+                  "createdTime": "2018-11-13T19:33:10.166Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 604800000,
+                  "createdTime": "2018-11-13T19:33:10.166Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 1209600000,
+                  "createdTime": "2018-11-13T19:33:10.166Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 2592000000,
+                  "createdTime": "2018-11-13T19:33:10.167Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                }
+              ],
+              "allowCustom": true
+            },
+            "timeContextFromParameter": null
+          },
+          {
+            "id": "3b5cc420-8ad8-4523-ba28-a54910756794",
+            "version": "KqlParameterItem/1.0",
+            "name": "Apps",
+            "type": 2,
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "query": "SigninLogs\r\n| summarize by AppDisplayName\r\n| order by AppDisplayName asc\r\n| project Value = AppDisplayName, Label = AppDisplayName, Selected = false\r\n| union (datatable(Value:string, Label:string, Selected:boolean)[\r\n'*', 'All Apps', true\r\n])",
+            "crossComponentResources": [
+              "{Workspace}"
+            ],
+            "isHiddenWhenLocked": false,
+            "timeContextFromParameter": null,
```

Comment:
you probably want to choose your time context param `TimeRange` here as well, so that this parameter query also uses the time range?  otherwise, this query is doing an implicit last 24 hours query.

Reply:
fixed it